public/js/reader.js

```diff
@@ -144,8 +152,30 @@ Reader.initializeAll = function () {
             Reader.loadImages();
         },
     );
+
+    // Fetch "bookmark" category ID and setup icon
+    Reader.loadBookmarkStatus();
 };
 
+/**
+ * Adds a removable category flag to the categories section within archive overview.
+ */
+Reader.addCategoryFlag = function ( categoryId ) {
```

Comment:
"Flag" made little sense to me when first reading the changes and then I understood it was just refactoring the existing subitems that go in archive overview.  

Can we rename this to  
```suggestion
Reader.addCategoryBadge = function ( categoryId ) {
```